### PR TITLE
Point at correct redcap variable for expected weight

### PR DESCRIPTION
**Pull request by aherbst-broad**
## Why

[Relevant ticket](https://broadinstitute.atlassian.net/browse/DSPDC-1458)

## This PR
* Points at the correct field name (verified by pulling down a JSON extract)

## TODO 
- [ ] Update mapping table?

### CODE REVIEW

File: hles/transformation/src/test/scala/org/broadinstitute/monster/dap/dog/DemographicsTransformationsSpec.scala [hunk 1]
```diff
@@ -391,7 +391,7 @@ class DemographicsTransformationsSpec extends AnyFlatSpec with Matchers with Opt
     val example = Map[String, Array[String]](
       "dd_dog_weight" -> Array("3"),
       "dd_dog_weight_lbs" -> Array("100.25"),
-      "dd_weight_range_expected_adult" -> Array("4")
+      "dd_dog_weight_as_adult" -> Array("4")
```
Comment (ThePletch) [inline hles/transformation/src/test/scala/org/broadinstitute/monster/dap/dog/DemographicsTransformationsSpec.scala:394]: it looks like the ticket spells this as `dd_dog_weigh_as_adult`, with no 't' in 'weight'.

**Review by ThePletch — CHANGES_REQUESTED** [review]
think we need to ensure we've got the right spelling for this variable

File: hles/transformation/src/test/scala/org/broadinstitute/monster/dap/dog/DemographicsTransformationsSpec.scala (hunk 1 above)
Comment (aherbst-broad) [reply hles/transformation/src/test/scala/org/broadinstitute/monster/dap/dog/DemographicsTransformationsSpec.scala:394]: @ThePletch The ticket is incorrect, grepping the latest extracted json shows data for `dd_dog_weight_as_adult`, and none for `dd_dog_weigh_as_adult`

Comment (aherbst-broad) [reply hles/transformation/src/test/scala/org/broadinstitute/monster/dap/dog/DemographicsTransformationsSpec.scala:394]: We've confirmed the `dd_dog_weight_as_adult` with the DAP team.

**Review by ThePletch — APPROVED** [review]
spelling is good, so let's ship it

**Review by quazi-h — APPROVED** [review]

**Review by eparkmonster — APPROVED** [review]

**Pull request merged**